### DISCUSSION TITLE
HELP: ao3_login() missing 1 required positional argument: 'fileops'

### DISCUSSION THREAD

**Discussion by pandamega** [nianeyna/ao3downloader#62] [Q&A] [answered]
I'm having trouble with the program, it was working fine, I changed some download folders, and now I'm getting this error right after it's asks about saving images, when it's supposed to ask me to log in.

`ao3_login() missing 1 required positional argument: 'fileops'`
after it just goes to "'d' to display the menu again or 'q' to quit"

I don't know what to do to fix the issue. I tried re-downloading the code and starting from scratch but it didn't make a difference. any help would be appreciated!

**Comment by nianeyna** [comment, marked as answer]
Oh man, totally my bad. I changed some code a bit ago and apparently I forgot to update a reference to it in the series update function. I just pushed a fix, so if you get a fresh copy of the code it should work now! Thanks so much for reporting this bug, I really appreciate it!
